addons/mail/static/src/owl/components/chat_window_manager/chat_window_manager.js

```diff
@@ -119,6 +119,50 @@ class ChatWindowManager extends Component {
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * Cycles to the next possible chat window starting from the
+     * `currentChatWindowLocalId`, following the natural order based on the
+     * current text direction, and with the possibility to `reverse` based on
+     * the given parameter.
+     *
+     * @private
+     * @param {integer} currentChatWindowLocalId
+     * @param {boolean} reverse
+     */
+    _cycleChatWindow(currentChatWindowLocalId, reverse = false) {
```

Comment:
prefer object for options, e.g. `{ reverse = false } = {}`

Reply:
ok